pyscript.core/test/terminal.py

```diff
@@ -0,0 +1,22 @@
+###### magic monkey patching ######
+import sys
+import builtins
+import js
+from pyscript import sync
+from pyodide.code import eval_code
+
+sys.stdout = sync
+builtins.input = sync.readline
+
+globals = {"js": js}
+
+####### main code ######
+while True:
+    code = input(f"> ")
+    if len(code):
+        try:
+            result = eval_code(f"{code}", globals=globals)
+            if result is not None:
+                print(result)
+        except:
+            print(f"Unable to evaluate: {code}")
```

Comment:
there is a better way to do it: simply do:
```python
import code
code.interact()
```

and you get a fully working python console

Reply:
done. We could bring this in for fun, it's a quick smoke test for a repl worth having around, imho.